### PR TITLE
Add new current events view

### PR DESCRIPTION
**Pull request by Andreslw**
Add empty events view
Add new current events view

solves #113  and #115

### CODE REVIEW

File: src/partner/modules/CardEvent/CardEvent.tsx [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+`;
+
+interface CardEventProps {
+  key: string;
+  eventInfo: IEvent.IEvent;
+}
+
+export const CardEvent: React.SFC<CardEventProps> = ({ eventInfo }) => {
+  const printProducts = () => {
+    return [1, 2, 3].map(e => {
```
Comment (mishelashala) [inline src/partner/modules/CardEvent/CardEvent.tsx]: load data from a service or repository, avoid hardcoding data on the view

File: src/partner/views/CurrentEvents/CurrentEventsContainer.tsx [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+import { EventList } from "src/partner/modules/EventList/EventList";
+import { ContentWrapper } from "src/partner/modules/ui";
+
+export interface CurrentEventsContainerProps {
+  events: IEvent[];
+}
+
+export const CurrentEventsContainer: React.SFC<CurrentEventsContainerProps> = ({ events = [] }) => {
+  return (
+    <>
```
Comment (mishelashala) [inline src/partner/views/CurrentEvents/CurrentEventsContainer.tsx]: Favor `React.Fragment` over `<>`

**Review by mishelashala — CHANGES_REQUESTED** [review]

File: src/partner/views/CurrentEvents/CurrentEventsContainer.tsx (hunk 2 above)
Comment (Andreslw) [reply src/partner/views/CurrentEvents/CurrentEventsContainer.tsx]: ok

**Comment by Andreslw** [comment]
Event requests are missing the "products" property, we need to add them.
Currently the client is working with the fake api 

Create:
Screenshot from 2019-04-17 16-16-28

Get all events: 
Screenshot from 2019-04-17 16-18-55

**Review by mishelashala — APPROVED** [review]

**Pull request merged**